doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -0,0 +1,465 @@
+    "\n",
+    "        # self.metric.compute() returns a dictionary:\n",
+    "        # e.g. {\"matthews_correlation\": 0.53}\n",
+    "        self.log_dict(matthews_correlation, sync_dist=True)\n",
+    "        self.predictions.clear()\n",
+    "        self.references.clear()\n",
+    "\n",
+    "    def configure_optimizers(self):\n",
+    "        return torch.optim.AdamW(self.parameters(), lr=self.lr, eps=self.eps)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Finetune the model with LightningTrainer\n",
+    "\n",
+    "Define a LightningTrainer with necessary configurations, including hyper-parameters, checkpointing and compute resources settings. \n",
+    "\n",
+    "You may find the API of {class}`LightningConfigBuilder <ray.train.lightning.LightningConfigBuilder>` useful.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ray.train.lightning import LightningTrainer, LightningConfigBuilder\n",
+    "from ray.air.config import RunConfig, ScalingConfig, CheckpointConfig\n",
+    "\n",
+    "# Define the configs for LightningTrainer\n",
+    "lightning_config = (\n",
+    "    LightningConfigBuilder()\n",
+    "    .module(cls=SentimentModel, lr=1e-5, eps=1e-8)\n",
+    "    .trainer(max_epochs=5, accelerator=\"gpu\")\n",
+    "    .checkpointing(save_on_train_epoch_end=False)\n",
+    "    .build()\n",
+    ")\n",
+    "\n",
+    "# Save AIR checkpoints according to the performance on validation set\n",
+    "run_config = RunConfig(\n",
+    "    name=\"ptl-sent-classification\",\n",
+    "    checkpoint_config=CheckpointConfig(\n",
+    "        num_to_keep=2,\n",
+    "        checkpoint_score_attribute=\"matthews_correlation\",\n",
+    "        checkpoint_score_order=\"max\",\n",
+    "    ),\n",
+    ")\n",
+    "\n",
+    "# Scale the training workload across 4 GPUs\n",
+    "# You can change this config based on your compute resources.\n",
+    "scaling_config = ScalingConfig(\n",
+    "    num_workers=4, use_gpu=True, resources_per_worker={\"CPU\": 1, \"GPU\": 1}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "if SMOKE_TEST:\n",
+    "    lightning_config = (\n",
+    "        LightningConfigBuilder()\n",
+    "        .module(cls=SentimentModel, lr=1e-5, eps=1e-8)\n",
+    "        .trainer(max_epochs=2, accelerator=\"gpu\")\n",
+    "        .checkpointing(save_on_train_epoch_end=False)\n",
+    "        .build()\n",
+    "    )\n",
+    "\n",
+    "    for split, ds in ray_datasets.items():\n",
+    "        ray_datasets[split] = ds.random_sample(0.1)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Train the model with the configuration we specified above. \n",
```

Comment:
I think you can split this into a new header. In particular, the content above this line can be "Configuring your LightningTrainer", and then the content below can be "Fine-tune the model with LightningTrainer".

With this, you can also fill out the section some more by describing `LightningConfigBuilder`, `RunConfig`, and `ScalingConfig` in more detail.